src/commands/land.rs

```diff
@@ -18,18 +18,25 @@ use crate::{
 };
 
 #[derive(Debug, clap::Parser)]
-pub struct LandOptions {}
+pub struct LandOptions {
+    /// Merge a Pull Request that was created or updated with spr diff
+    /// --cherry-pick
+    #[clap(long)]
+    cherry_pick: bool,
+}
 
 pub async fn land(
-    _opts: LandOptions,
+    opts: LandOptions,
     git: &crate::git::Git,
     gh: &mut crate::github::GitHub,
     config: &crate::config::Config,
 ) -> Result<()> {
     git.check_no_uncommitted_changes()?;
     let mut prepared_commits = git.get_prepared_commits(config)?;
 
-    if prepared_commits.len() > 1 {
+    let based_on_unlanded_commits = prepared_commits.len() > 1;
+
+    if based_on_unlanded_commits && !opts.cherry_pick {
         return Err(Error::new(formatdoc!(
             "Cannot land a commit whose parent is not on {master}. To land \
              this commit, rebase it so that it is a direct child of {master}.",
```

Comment:
should this message mention the --cherry-pick flag?

Reply:
:+1: